system/Boot.php

```diff
@@ -0,0 +1,229 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+namespace CodeIgniter;
+
+use CodeIgniter\Config\DotEnv;
+use CodeIgniter\Exceptions\FrameworkException;
+use Config\Autoload;
+use Config\Modules;
+use Config\Paths;
+use Config\Services;
+
+/**
+ * Bootstrap for the application
+ *
+ * @codeCoverageIgnore
+ */
+class Boot
+{
+    /**
+     * @used-by public/index.php
```

Comment:
The usage of `@used-by` is maybe not correct because `public/index.php` is not a Structural Element.
https://docs.phpdoc.org/guide/references/phpdoc/tags/uses.html#uses-used-by

Reply:
I removed the tags, and made it just a comment.
Because both tags need a Structural Element Name, but a file is not a Structural Element.